### PR TITLE
Adds CLI middleware reject/resolve tests

### PR DESCRIPTION
**Pull request by bennett000**
These tests aren't elaborate they just ensure that API calls made by the CLI middleware get handled.

### CODE REVIEW

File: src/api/0.1/cli/cli-authorities.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const cn = require('../js/js-api');
+let cn = require('../js/js-api');
```
Comment (SethDavenport) [inline src/api/0.1/cli/cli-authorities.js]: Why let insyead of const?

**Comment by bennett000** [comment]
@SethDavenport `cn` (clusternator API) gets mocked out in the test and `const` prevents the mocking.

File: src/api/0.1/cli/cloud-service.spec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Q = require('q');
+const rewire = require('rewire');
+
+const c = rewire('./cloud-service');
+const C = require('../../../chai');
+
+/*global describe, it, expect, beforeEach, afterEach */
```
Comment (SethDavenport) [inline src/api/0.1/cli/cloud-service.spec.js]: Is this 'global' comment for `eslint`?  I feel like I've put that tyoe of thing in `eslintrc` before.

Comment (bennett000) [inline src/api/0.1/cli/cloud-service.spec.js]: For both eslint, and my local jshint.  I will get those migrated to eslintrc

File: src/api/0.1/cli/cli-authorities.js (hunk 1 above)
Comment (SethDavenport) [inline src/api/0.1/cli/cli-authorities.js]: Oh right... rewire. Nvm

**Pull request merged**